templates/zerver/integrations/mastodon.md

```diff
@@ -0,0 +1,39 @@
+Fetch public posts (sometimes called "toots") of individual accounts or
+federated hashtags from Mastodon in Zulip! While Zulip doesn't integrate
+directly with ActivityPub or the overall Fediverse, some ActivityPub servers,
+like [Mastodon](https://joinmastodon.org/), publish various RSS feeds which can
+be followed in Zulip using the [RSS][1] or [Zapier][2] integrations.
+
+!!! tip ""
+
+    Note that due to the various interactions and complexities of federation,
+    following a hashtag on one homeserver does not guarantee all posts in the
+    entire Fediverse will be included in the feed, only that all public posts
+    that have federated to the instance in question will be included. For a
+    visual guide to how Mastodon's Public Timeline works, [see this Reddit
+    comment with an included
+    flowchart](https://www.reddit.com/r/Mastodon/comments/8if9r6/adding_other_mastodon_instance_feeds_to_my/dyrjnq4/).
+
+    Inherently, this means that `https://fosstodon.org/tags/zulip.rss` and, for
+    example, `https://personal.mastodon.server.example.com/tags/zulip.rss` may
+    have vastly different contents depending on which servers
+    `personal.mastodon.server.example.com` has interacted with or in some way
+    moderated.
+
+1. Find the RSS feed for the entity you'd like to follow, often by appending
+   `.rss` to its existing Mastodon URL. For example, to follow the public posts
```

Comment:
Maybe:

Find the RSS feed for the entity you'd like to follow. You can generally do this by appending `.rss` to the Mastodon URL for the feed you are interested in.

!!! tip ""

For example, to follow Zulip's Mastodon account at `https://fosstodon.org/@zulip`, you would use `https://fosstodon.org/@zulip.rss`. To follow the `#zulip` hashtag at `https://fosstodon.org/tags/zulip`, you would use  `https://fosstodon.org/tags/zulip.rss`.